lusid-sdk-python/tests/finbournetest.py

```diff
@@ -431,6 +442,154 @@ def create_property_definition_test(self, property_request, property):
+                    self.assertEqual(difference_cost_amount, round(rec_break.difference_cost.amount, 2))
+                    self.assertEqual(left_holding_units, round(rec_break.left_units, 0))
+                    self.assertEqual(left_holding_cost_amount, round(rec_break.left_cost.amount, 2))
+                    self.assertEqual(left_holding_cost_currency, rec_break.left_cost.currency)
+                    self.assertEqual(right_holding_units, round(rec_break.right_units,0))
+                    self.assertEqual(right_holding_cost_amount, round(rec_break.right_cost.amount,2))
+                    self.assertEqual(right_holding_cost_currency, rec_break.right_cost.currency)
+
+            # If there is no matching holding on the right, check that the reconciliation break matches the left side
+            except KeyError:
+                rec_break = rec_breaks[key]
+                self.assertEqual(left_holding_units, round(rec_break.difference_units, 0))
+                self.assertEqual(left_holding_cost_amount, round(rec_break.difference_cost.amount,2))
+                self.assertEqual(left_holding_cost_currency, rec_break.difference_cost.currency)
+                self.assertEqual(left_holding_units, round(rec_break.left_units,0))
+                self.assertEqual(left_holding_cost_amount, round(rec_break.left_cost.amount,2))
+                self.assertEqual(left_holding_cost_currency, rec_break.left_cost.currency)
+
+    def add_transaction_property_asserts(self, scope, code, transaction_id, property_key, property_value):
+        """
+        This method contains a set of assertions which test that a property has been successfully added to a transaction
+
+        :param scope: The scope of the portfolio that contains the transaction
+        :param code: The code of the portfolio that contains the transaction
+        :param transaction_id: The transaction id of the transaction
+        :param property_key: The property key of the property that we want to check
+        :param property_value: The property value of the property that we want to check
+        """
+        # Get our transaction by filtering on the transaction id across all transactions inside our portfolio
+        transaction = self.client.get_transactions(scope,
+                                                   code,
+                                                   filter="transactionId eq '{}'".format(transaction_id)).values[0]
+
+        # Create a dictionary with the key, value pairs for each of our properties on the transaction
+        transaction_properties = {trans_property.key:trans_property.value for trans_property in transaction.properties}
+
+        # Check that the property exists and that the value is correct
+        self.assertTrue(property_key in transaction_properties)
+        self.assertEqual(property_value, transaction_properties[property_key])
+
     def test(self):
```

Comment:
why is this needed, can it be removed?

Reply:
Yes it can be removed. I thought it was necessary for things to work correctly, but apparently not. I'll double check using Travis removing this has not broken anything.